qa/rspec/commands/ubuntu.rb

```diff
@@ -0,0 +1,17 @@
+# encoding: utf-8
+require_relative "debian"
+
+module ServiceTester
+  class UbuntuCommands < DebianCommands
+
+    def running?(hosts, package)
+      stdout = ""
+      at(hosts, {in: :serial}) do |host|
+        cmd = sudo_exec!("service #{package} status")
+        stdout = cmd.stdout
+      end
+      stdout.match(/^#{package} start\/running/)
```

Comment:
This will probably have to be updated to work with systemd style, as 16.04 uses that.  I can see that debian 8 uses systemd, so perhaps we can piggy-back on that.